packages/core/handlers/routers/auth.js

```diff
@@ -1,26 +1,36 @@
 const { createIntegrationRouter } = require('@friggframework/core');
 const { createAppHandler } = require('./../app-handler-helpers');
-const { requireLoggedInUser } = require('./middleware/requireLoggedInUser');
 const {
-    moduleFactory,
-    integrationFactory,
-    IntegrationHelper,
+    loadAppDefinition,
 } = require('./../backend-utils');
+const { UserRepository } = require('../../user/user-repository');
+const { IntegrationFactory, IntegrationHelper } = require('../../integrations/integration-factory');
+const { GetUserFromBearerToken } = require('../../user/use-cases/get-user-from-bearer-token');
+
+const { integrations, userConfig } = loadAppDefinition();
+const userFactory = new UserFactory(userConfig);
+const integrationFactory = new IntegrationFactory(integrations);
+const userRepository = new UserRepository({ userFactory });
+const getUserFromBearerToken = new GetUserFromBearerToken({ userRepository });
 
 const router = createIntegrationRouter({
-    factory: { moduleFactory, integrationFactory, IntegrationHelper },
-    requireLoggedInUser,
-    getUserId: (req) => req.user.getUserId(),
+    factory: {
+        moduleFactory: integrationFactory.moduleFactory,
+        integrationFactory,
+        IntegrationHelper
+    },
+    getUserFromBearerToken,
 });
 
+//todo: what is this route doing here?
 router.route('/redirect/:appId').get((req, res) => {
     res.redirect(
-        `${process.env.FRONTEND_URI}/redirect/${
-            req.params.appId
+        `${process.env.FRONTEND_URI}/redirect/${req.params.appId
         }?${new URLSearchParams(req.query)}`
     );
 });
 
 const handler = createAppHandler('HTTP Event: Auth', router);
 
+// todo: I can not find where router is used, do we need to export it?
 module.exports = { handler, router };
```

Comment:
Maybe check the canva repo?

I can't recall what prompted keeping this as an export.

Reply:
@seanspeaks My question was more on the "shouldn't this live somewhere else ?" direction.